crates/bifrost/src/providers/replicated_loglet/remote_sequencer.rs

```diff
@@ -481,7 +505,7 @@ mod test {
                 header: CommonResponseHeader {
                     known_global_tail: None,
                     sealed: Some(false),
-                    status: self.reply_status,
+                    status: self.reply_status.clone(),
```

Comment:
Is this needed?

Reply:
Yes, since SequencerStatus is no longer `Copy` because now it has the variant `Unknown(String)`

Reply:
But I am wondering if we should expand the Unknow(string) to an inclusive list of status variants